forge-ai/src/main/java/forge/ai/ability/CountersMoveAi.java

```diff
@@ -426,11 +426,9 @@ private AiAbilityDecision moveTgtAI(final Player ai, final SpellAbility sa) {
             // try to move to something useless or something that would leave play
             boolean isNegative = ComputerUtil.isNegativeCounter(cType, src);
             List<Card> filteredTgtList;
-            if (isNegative) {
-                filteredTgtList = CardLists.filterControlledBy(tgtCards, ai.getOpponents());
-            } else {
-                filteredTgtList = CardLists.filter(tgtCards, CardPredicates.isControlledByAnyOf(ai.getAllies()).or(CardPredicates.isController(ai)));
-            }
+            filteredTgtList = isNegative ? CardLists.filterControlledBy(tgtCards, ai.getOpponents()) :
+                CardLists.filter(tgtCards, CardPredicates.isControlledByAnyOf(ai.getAllies()).or(CardPredicates.isController(ai)));
```

Comment:
@Agetian maybe use `getYourTeam` ?
Or is getAllies + You != getYourTeam ? 🤔

Reply:
Good idea, thanks! :) Done, will commit a followup.